src/port/dr_audio.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+//*****************************************************************************
+// Copyright 2025 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma warning(push)
+#pragma warning(disable : 4245 4220)
+#include "dr_wav.h"  // NOLINT
```

Comment:
If there would be places where we would include just one not the other then split this into separate headers. Otherwise it could be left as is.